src/v/redpanda/application.cc

```diff
@@ -1083,6 +1086,7 @@ void application::start(::stop_signal& app_signal) {
 
 void application::start_redpanda() {
     syschecks::systemd_message("Staring storage services").get();
+    storage_node->start().get0();
```

Comment:
This is the common pattern here, but IIUC, there are a bunch of double calls to `service::start()`. I have an item on my TODO list to look into that and clean up if needed.

Reply:
> This is the common pattern here, but IIUC, there are a bunch of double calls to service::start(). I have an item on my TODO list to look into that and clean up if needed.

what's an example of a "double call to service::start"?

Reply:
The `construct_service()` helper in application.h calls start(). Then later on, people explicitly call it again. Look for `_admin` in application.cc, for example.

Reply:
Do you think you might be seeing a call to `ss::sharded<T>::start` following later by called to `ss::sharded<T>::invoke_on_all(T::start)`? The former call starts the service, where as the later call invokes `T::start` on each core which is defined but the author of `T`.